src/plugins/intel_cpu/tests/functional/custom/single_layer_tests/psroi_pooling.cpp

```diff
@@ -107,6 +107,14 @@ class PSROIPoolingLayerCPUTest : public testing::WithParamInterface<PSROIPooling
         const ov::ResultVector results{std::make_shared<ov::op::v0::Result>(psroi)};
         function = std::make_shared<ov::Model>(results, params, "PSROIPooling");
     }
+
+#if defined(OPENVINO_ARCH_RISCV64)
+private:
+    std::string getPrimitiveType() {
+         // TODO: Support psroi_pooling op for RISCV64 arch
+        return "ref";
+    }
+#endif
```

Comment:
I have firstly tried adding `getPrimitiveType()` for [Activation](https://github.com/a-sidorova/openvino/blob/9d0e61156d62cd9a38d8306fc52bf4270539ade7/src/plugins/intel_cpu/tests/functional/custom/single_layer_tests/classes/activation.cpp#L199) and [Eltwise](https://github.com/a-sidorova/openvino/blob/9d0e61156d62cd9a38d8306fc52bf4270539ade7/src/plugins/intel_cpu/tests/functional/custom/single_layer_tests/classes/eltwise.cpp#L270) test classes, and they worked fine if I launched these tests only. However, when I launched the rest of the tests, lots of tests failed due to the mismatch of primitive types.

I find the reason is that, I have enabled `shl` primitive type [here](https://github.com/BHbean/openvino/blob/shl-executor/src/plugins/intel_cpu/tests/functional/utils/cpu_test_utils.cpp#L269-L272) for all the CPU tests (take `ACL` as a reference), and if certain ops are not supported by our executor, it will prompt error like: `expected type: shl_f32, actual: f32` when testing. That is why I added so many extra `getPrimitiveType()` to make sure the right primitive type.